iputil/ip.go

```diff
@@ -0,0 +1,70 @@
+// Package iputil defines type that will marshal/unmarshal an IP address as
+// human readable IPv4 string rather than an unreadable byte stream.
+package iputil
+
+import (
+	"bytes"
+	"database/sql/driver"
+	"encoding/json"
+	"fmt"
+	"net"
+)
+
+// IP is a special wrapped type of net.IP
+type IP net.IP
+
+// Gt will return if the IP is greater than the given IP
+func (ip IP) Gt(start net.IP) bool {
+	return bytes.Compare(ip, start) >= 0
```

Comment:
The `>=` makes it look like it's "greater or equal than"? Same for `Lt()`.

Reply:
Good point, I'll test it and clarify that the behaviour includes equality and if it does, name it appropriately.